ember-client/shared/src/main/scala/org/http4s/ember/client/internal/EmberWSClient.scala

```diff
@@ -45,10 +45,21 @@ private[client] object EmberWSClient {
     SecureRandom.javaSecuritySecureRandom[F].map { random =>
       WSClient[F](respondToPings = false) { wsRequest =>
         for {
+          uriScheme <- wsRequest.uri.scheme
+            .liftTo[F](new RuntimeException("URI scheme missing"))
```

Comment:
I don't think we need to raise an error here. If there is no scheme in the URI we can just leave it.